Fix Female Undergarment Clipping

Bonaout opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Changes the binder and tanktop undershirt markings to have a version for each sex, to fix the clipping issue that has been bugging me for a long time

I did something with code... I don't know if it's good

The clipping still appears in the character customization menu, but works in game!

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Bugfix

## Technical details
<!-- Summary of code changes for easier review. -->


## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="898" height="421" alt="Capture d&#39;écran 2025-11-01 155645" src="https://github.com/user-attachments/assets/8896cbac-9912-4e5d-a647-9abb0af7f15e" />

<img width="898" height="421" alt="Capture d&#39;écran 2025-11-01 150950" src="https://github.com/user-attachments/assets/26ead317-3b61-4acc-a0e3-e714ccf566f2" />


## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- fix: Undergarment markings now correctly apply female displacements!

Bonaout committed:
Fix Female Undergarment Clipping

Fixes the undergament clipping issues due to the female displacement maps by making female and unsexed versions of the tanktop and binder

Bonaout committed:
Revert "Fix Female Undergarment Clipping"

This reverts commit afcec266f3a303fe5257d2e9baf908322d4a332f.

Absotively commented:
From a very brief investigation, it looks to me like adding displacement maps to the markingsDisplacement value of various species' HumanoidAppearance components would also work, and would apply the displacement automatically to many things. But it also doesn't appear to support applying maps to only specific genders :thinking:.

Bonaout committed:
I did a code thing

I made a female only markings displacement component something... I'm not sure, I'm tired... But it works!

Bonaout commented:
> I think it probably needs to use the sex-specific displacement _instead of_ the generic displacement, rather than using both of them. Trying to use both doesn't cause problems when only one exists, but I think both will need to exist for moths. And possibly downstream species.

I'm sorry, but I don't really understand what you mean?

Bonaout committed:
Boop

Bonaout commented:
Thanks for the dummy/character editor tip !! I was wondering why it wasn't applying there...

As for the multiple displacement maps stacking with certain species... I will try to do some testing later, but since this only applies to markings, specifically the markings categories we choose to apply them to, I am hopeful that there won't be any issues!

Bonaout committed:
Displacements in character editor

Apply the displacement maps changes to the species dummies

TheShuEd commented:
<img width="533" height="651" alt="image" src="https://github.com/user-attachments/assets/9f01fd41-d3c4-4463-ad2c-e5cd649acd07" />

This PR is heavily outdated due Nubody refactor

TheShuEd commented:
thanks for you contribution anyway!
